cdap-ui/app/cdap/components/AbstractWidget/FormInputs/TextBox/index.tsx

```diff
@@ -23,6 +24,7 @@ import { objectQuery } from 'services/helpers';
 interface ITextBoxWidgetProps {
   placeholder?: string;
   default?: string;
+  enableUnderline?: boolean; // TODO Add to all widgets
```

Comment:
nit: Can we have a JIRA for this TODO with a fix version? Otherwise we will seldom revisit this issue.

Reply:
Yup, it's here: https://cdap.atlassian.net/browse/CDAP-17244